pkgs/applications/networking/mailreaders/thunderbird/wrapper.nix

```diff
@@ -16,7 +16,7 @@ args:
   # For that to work out of the box, it requires `gnupg` on PATH and
   # `gpgme` in `LD_LIBRARY_PATH`; we do this below.
   buildCommand = old.buildCommand + ''
-    wrapProgram $out/bin/thunderbird \
+    wrapProgram $out/bin/${browser.binaryName} \
```

Comment:
Right now after this PR was merged, `thunderbird-bin` fails to eval because it doesn't have `binaryName`

I don't know why this didn't get caught as an eval error, but maybe this should use `applicationName` like in `pkgs/applications/networking/browsers/firefox/wrapper.nix`?